webpack/storybook/atoms/UploaderWidget.stories.js

```
@@ -0,0 +1,138 @@
+  variant: "uploadWidget",
+  multiple: true,
+  validations: {
+    maxSizeAllowedInMB: 5,
+    minSizeRequiredInMB: 1,
+  },
+  showErrorCard: true,
+  showDownloadButton: true,
+  showReUploadButton: true,
+};
+
+export const Error = Template.bind({});
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)